package.json

```diff
@@ -83,5 +83,8 @@
   },
   "peerDependencies": {
     "pixi.js": "^4.3.4"
+  },
+  "dependencies": {
+    "checkpack": "^0.2.7"
```

Comment:
This wasn't included in the package.json because **checkpack** is a rather weighty dependency. Instead, it's installed in the `preintegrations` script. This is only run on Travis.

Reply:
I see 👍 
I delete `3711ed217e193af216b2c70e566a61958c62a5e9`.